pkg/typedef/types.go

```diff
@@ -67,7 +67,10 @@ var (
 	TypesMapKeyBlacklist = map[SimpleType]struct{}{
 		TypeBlob:     {},
 		TypeDuration: {},
+		TypeVarint:   {},
```

Comment:
would be good to add clues why (e.g. simple explanation or linking to the issue)